### DISCUSSION TITLE
troubleshoot with Plug N Meet

### DISCUSSION THREAD

**Discussion by ArtesadeSegre5** [mynaparrot/plugNmeet-server#363] [Q&A]
Hello everyone
I have a problem being able to share the audio of a Zoom video conference using Mac in full screen in Plug N Meet. It only gives me the option to share audio itself if I share Zoom as a Google Chrome tab, which is impossible for me because when I use Zoom it Open the application (which is why I need to share full screen but it does not enable the audio sharing option in Plug N Meet. Can someone please help me. Thank you very much

**Comment by jibon57** [comment]
That's not possible unless otherwise Google chrome support capturing sound from application. Even sharing tab sound sometime doesn't work because of various bug. As plugNmeet is a browser based application, we can support whatever browser is supporting so far.

**Reply by jibon57** [reply]
In this case you can use [OBS](https://obsproject.com/) to capture the screen with sound of the application & either use virtual camera (from obs) or ingress feature from plugNmeet to stream from OBS.

**Reply by ArtesadeSegre5** [reply]
Thank you very much. I am a psychologist and teacher in family therapy and I need a platform that allows me, on the one hand, to project a live therapy session on a screen or videoconference, and on the other hand, to share what is happening in the session on another screen where my students have access. of therapy. What platform do you recommend? What technical requirements should it have? If you can help me I would appreciate it very much
